sdk/script/management-sdk.ts

```diff
@@ -208,7 +195,17 @@ export class AccountManager {
             .then(() => null);
     }
 
-    public releasePackage(appName: string, deploymentName: string, fileOrPath: File | string, description: string, appVersion: string, isMandatory: boolean = false, uploadProgressCallback?: (progress: number) => void): Promise<void> {
+    public getDeploymentMetrics(appName: string, deploymentName: string): Promise<DeploymentMetrics> {
+        return this.get(urlEncode `/apps/${appName}/deployments/${deploymentName}/metrics`)
+            .then((res: JsonResponse) => res.body.metrics);
+    }
+
+    public getDeploymentHistory(appName: string, deploymentName: string): Promise<Package[]> {
+        return this.get(urlEncode `/apps/${appName}/deployments/${deploymentName}/history`)
+            .then((res: JsonResponse) => res.body.history);
+    }
+
+    public release(appName: string, deploymentName: string, fileOrPath: File | string, description: string, appVersion: string, isMandatory: boolean = false, uploadProgressCallback?: (progress: number) => void): Promise<void> {
```

Comment:
Should we move "description" to the right of "targetBinaryVersion" and make it optional?